service/awsconfig/v7/resource/cloudformation/current.go

```diff
@@ -87,7 +87,7 @@ func (r *Resource) GetCurrentState(ctx context.Context, obj interface{}) (interf
 			return StackState{}, microerror.Mask(err)
 		}
 		versionBundleVersion, err := r.service.GetOutputValue(stackOutputs, key.VersionBundleVersionKey)
-		if IsNotFound(err) {
+		if cloudformationservice.IsOutputNotFound(err) {
```

Comment:
This is a bug apparently. When adding the service abstraction I forgot to align the error matcher.